dags/dag_content/tower_metrics_content.py

```diff
@@ -0,0 +1,411 @@
+import os
+import json
+import time
+
+from airflow.decorators import task
+from airflow.models import Variable
+import boto3
+import synapseclient
+
+#VARIABLES
+
+# AWS creds
+AWS_CREDS = {
+    "AWS_ACCESS_KEY_ID": Variable.get(
+        "TOWER_DB_ACCESS_KEY"
+    ),
+    "AWS_SECRET_ACCESS_KEY": Variable.get(
+        "TOWER_DB_SECRET_ACCESS_KEY"
+    )
+}
+#AWS region
+AWS_REGION = "us-east-1"
+
+# cluster names
+DATABASE_NAME = "tower"
+CLONE_DATABASE_NAME = "tower-clone"
+
+# query dictionary - maps query names to queries
+QUERY_DICT = {
```

Comment:
I think this is fine for now, but we may want this to live somewhere else in the future